tests/unit/vic_run/test_snow_utility.py

```diff
@@ -2,7 +2,8 @@
 from vic.vic import ffi
 from vic import lib as vic_lib
 
-
+# TODO: design new test to not only use the static parameter value
+@pytest.mark.xfail
```

Comment:
this should be fixed before we merge this.

Reply:
Agreed

Reply:
OK - we'll fix the test in future PR